src/components/Stepper/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import Button from "../Button";
+import Icon from "../Icon";
+
+const Stepper = ({
+  steps,
+  isDisableFinish = false,
+  onFinish = () => {},
+  onStepChange = () => {},
+  finishBtnText = "Finish",
+}) => {
+  const [currentStep, setCurrentStep] = useState(0);
+
+  useEffect(() => {
+    const initialStep = steps.findIndex((step) => step.isInitial);
+    setCurrentStep(initialStep !== -1 ? initialStep : 0);
+  }, [steps]);
+
+  const handleNext = () => {
+    if (currentStep < steps.length - 1) {
+      const nextStep = currentStep + 1;
+      setCurrentStep(nextStep);
+      onStepChange(steps[nextStep]);
+    }
+  };
+
+  const handleBack = () => {
+    if (currentStep > 0) {
+      const prevStep = currentStep - 1;
+      setCurrentStep(prevStep);
+      onStepChange(steps[prevStep]);
+    }
+  };
+
+  const isLastStep = currentStep === steps.length - 1;
+
+  return (
+    <div className="Stepper-container">
+      <div className="steps-container">
+        {steps.map((step, index) => {
+          const isStepComplete = index < currentStep;
+          return (
+            <div
+              key={index}
+              className={`step ${index === currentStep ? "active" : ""} ${
+                isStepComplete ? "completed" : ""
+              }`}
```

Comment:
I would write it in a different way to avoid whitespaces in the class attribute and maybe make it a little more clearer.
```suggestion
              className={[
                'step', 
                index === currentStep && 'active', 
                isStepComplete && 'completed'
              ].filter(Boolean).join(' ')}
```